src/qmoperators/two_electron/ExchangePotentialD1.cpp

```diff
@@ -53,9 +53,9 @@ void ExchangePotentialD1::setupInternal(double prec) {
             Orbital &phi_i = iter.orbital(0);
             int idx = iter.idx(0);
             for (int j = 0; j < Phi.size(); j++) {
+                if (mpi::my_orb(phi_i) and j <= idx) continue; // compute only i<j for own block
```

Comment:
My intention here was to disable the ij-ji symmetry.

Reply:
why?
Your version did not work in parallel. But if you really want to double the work, I can try to implement it.